sdk/core/azure-core-tracing-opencensus/azure/core/tracing/ext/opencensus_span/__init__.py

```diff
@@ -86,7 +86,10 @@ def kind(self, value):
         """Set the span kind of this span."""
         kind = (
             OpenCensusSpanKind.CLIENT if value == SpanKind.CLIENT else
+            OpenCensusSpanKind.CLIENT if value == SpanKind.PRODUCER else  # No producer in opencensus
```

Comment:
Nitpick: I would probably have considered using a `dict` instead of multiple `if`/`else`s. But this is probably faster given the relatively small number of items we have to map...

Reply:
Fixed